src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/settingsComponents/GeneralFeedback/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { injectIntl, FormattedMessage, intlShape } from '@edx/frontend-platform/i18n';
+import { Form } from '@edx/paragon';
+import PropTypes from 'prop-types';
+import SettingsOption from '../../SettingsOption';
+import messages from './messages';
+import { generalFeedbackHooks } from './hooks';
+
+export const GeneralFeedbackCard = ({
+  generalFeedback,
+  updateSettings,
+  // inject
+  intl,
+}) => {
+  const { summary, handleChange } = generalFeedbackHooks(generalFeedback, updateSettings);
+  return (
+    <SettingsOption
+      title={intl.formatMessage(messages.generalFeebackSettingTitle)}
+      summary={intl.formatMessage(summary.message, { ...summary.values })}
```

Comment:
This is invalid when i put in some text. This is what I get when I add some feedback and collapse the section.
<img width="321" alt="Screenshot 2023-02-07 at 11 33 50 AM" src="https://user-images.githubusercontent.com/42981026/217305580-93c89c56-ed6b-48eb-8d29-f57e265e2c95.png">